app/src/app/components/contents/PacketGroup/PacketGroup.tsx

```diff
@@ -1,19 +1,48 @@
-import { useParams } from "react-router-dom";
 import { PacketTable } from "./PacketTable";
+import {useParams} from "react-router-dom";
+import {useGetPacketGroupSummaries} from "../explorer/hooks/useGetPacketGroupSummaries";
+import {HttpStatus} from "../../../../lib/types/HttpStatus";
+import {Unauthorized} from "../common/Unauthorized";
+import {ErrorComponent} from "../common/ErrorComponent";
+import {useGetPacketById} from "../common/hooks/useGetPacketById";
 
 export const PacketGroup = () => {
   const { packetName } = useParams();
+  const { packetGroupSummaries, error: packetGroupError } =
+    useGetPacketGroupSummaries(0, 1, packetName as string);
+  const latestPacketId = packetGroupSummaries?.content[0]?.latestId;
```

Comment:
also just realised since display name is packet specific it doesn't make sense to show it at the top? since not all packets will have the same display name?

Reply:
We're going to show the display name of the most recent packet.